woodpecker/woodpecker.go

```diff
@@ -291,12 +291,12 @@ func New(c Config, stdout, stderr *log.Logger, clk clock.Clock) (*Woodpecker, er
 		}
 		cert, err := pki.LoadCertificate(c.SubmitConfig.CertIssuerPath)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("loading issuer certificate from %s: %s", c.SubmitConfig.CertIssuerPath, err)
```

Comment:
When printing filenames in error messages I think `%q` is a better format specifier than `%s` because it makes the most common failure mode (not having specified a file) easier to spot as `loading issuer certificate from "": error error error` instead of `loading issuer certificate from : error error error`.


```suggestion
			return nil, fmt.Errorf("loading issuer certificate from %q: %s", c.SubmitConfig.CertIssuerPath, err)
```

Reply:
Good call, thanks! I just learned about %q right after creating this PR.  I pushed a change.